styleguide/source/_patterns/01-molecules/06-components/15-table/_table.scss

```diff
@@ -0,0 +1,58 @@
+/**
+ * @file accordion.scss
+ *
+ * Copyright 2017 Palantir.net, Inc.
```

Comment:
@rpama Follow up that only Palantiri need to add the Palantir copyright notice when creating a file, and there is an open question on how to improve the process with a blanket statement.